packages/core-react/package.json

```
@@ -9,21 +9,26 @@
     "dev": "yarn storybook",
     "build": "gulp scripts:es scripts:umd scripts:compiled",
     "test": "jest",
+    "tdd": "jest --watch",
```

```suggestion
    "test:watch": "jest --watch",
```

maybe a bit more clear, not really sure if this is necessary

---

i personally prefer typing less :-P